__tests__/api.test.js

```diff
@@ -102,5 +109,45 @@ describe('GET /api/articles', () => {
                 })
             })
     })
-  
+
+})
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200: responds with an array of comments for the given article_id', () => {
+        return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.comments.length >= 1)
+                expect(body.comments).toBeSortedBy('created_at', {
+                    descending: true
+                })
+                body.comments.forEach((comment) => {
+                    expect(comment).toMatchObject({
+                        created_at: expect.any(String),
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        article_id: expect.any(Number)
+                    })
+                })
+            })
+    })
+    test('404: responds with appropriate status and msg when given a valid id but non-existent id', () => {
+        return request(app)
+        .get("/api/articles/999/comments")
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe('article does not exist')
+        });
+    })
+    test('400: sends and appropriate status and error message when given an ivalid id', () => {
+        return request(app)
+            .get("/api/articles/banana/comments")
+            .expect(400)
+            .then(({ body }) => {
+                expect(body.msg).toBe('Bad request');
+            });
+    });
```

Comment:
Another test you could add would be 
Status 200 - valid ID but no comments so responds with an empty array